### PR TITLE
Implement new OVO flow

### PR DESCRIPTION
**Pull request by hakiemaul**

### CODE REVIEW

**Comment by candrasaputra** [comment]
Where code to check order status frequently?

File: Xendit/M2Invoice/Controller/Checkout/Notification.m22.php [hunk 1]
```diff
@@ -60,13 +60,13 @@ public function execute()
             if (!empty($decodedPost['ewallet_type'])) {
                 $isEwallet = true;
 
-                if (!$decodedPost['external_id']) {
+                if (!$decodedPost['id']) {
```
Comment (candrasaputra) [inline Xendit/M2Invoice/Controller/Checkout/Notification.m22.php:63]: why change to id not keep using external_id?
it's same, right. or external_id will delete?

Comment (hakiemaul) [reply Xendit/M2Invoice/Controller/Checkout/Notification.m22.php:63]: oops yea my bad

**Comment by hakiemaul** [comment]
> Where code to check order status frequently?

https://github.com/xendit/xendit-magento-payment-module/pull/22/files#diff-3ce92d79eae49d8b4f214acf11683969R48-R56

File: Xendit/M2Invoice/Model/Payment/AbstractInvoice.php [hunk 2]
```diff
@@ -116,9 +116,9 @@ public function isAvailable(\Magento\Quote\Api\Data\CartInterface $quote = null)
                 return true;
             }
             //uncomment this block for local testing
-            /*if($this->methodCode == 'OVO'){
+            if($this->methodCode == 'OVO'){
                 return true;
-            }*/
+            }
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Model/Payment/AbstractInvoice.php:121]: comment back

**Pull request merged**